packages/wallets/src/components/WalletText/WalletText.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
would love to see screenshots from component on PR description :) make it way easier to review PRs